src/PKSim.Core/Snapshots/ExtendedProperties.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace PKSim.Core.Snapshots
+{
+   public class ExtendedProperties : SnapshotBase
+   {
+      public List<ExtendedProperty> ListOfExtendedProperties { set; get; }
```

Comment:
Look at how the CalculationMethodCache is defined and serialized. You do not need to have a sub properties here. the json will just be complicated for nothing. Instead just defined your type ExtendedProperties as IEnuerable

Or remove that type alltogether. It really does not hold anything aside from the list.